scripts/rejected-payloads.md

```diff
@@ -0,0 +1,87 @@
+## What is this
+
+rejected-payload is a python script used to categorize the reject-reason column of the release_tags table of the Sippy DB.
+reject-reason column is used to indicate, for a particular rejected payload, why it was rejected.
+This is intended to be used by the watcher as he monitors the status of each payload.
+The goal is to provide us some statistics of payload rejection history.
+
+Here is the list of known categories:
+
+```
+* "TEST_FLAKE",            # Payload was rejected because of flake tests
+* "CLOUD_INFRA",           # Payload was rejected because of infrastructure failures from cloud providers
+* "RH_INFRA",              # Payload was rejected because of other infrastructure failures
+* "PRODUCT_REGRESSION",    # Payload was rejected because of real product regression
+* "TEST_REGRESSION"        # Payload was rejected because of a test regression (typically from origin repo)
+```
+
+## Python Dependencies
+
+Use pip to install the following dependencies:
+
+`pip install argparse sqlalchemy`
```

Comment:
WDYT of `sudo dnf install python3-sqlalchemy+postgresql` on Fedora? Personally I prefer not to pip if there's OS packages available.